R/export.R

```diff
@@ -10,8 +10,11 @@
 #'   `shinylive.quiet` option or defaults to `FALSE` in interactive sessions if
 #'   not set.
 #' @param verbose Deprecated, please use `quiet` instead.
-#' @param wasm_packages Download and include binary WebAssembly packages as
-#'   part of the output app's static assets. Defaults to `TRUE`.
+#' @param wasm_packages Download and include binary WebAssembly packages as part
+#'   of the output app's static assets. Logical, defaults to `TRUE`. The default
+#'   value can be changed by setting the environment variable
+#'   `SHINYLIVE_WASM_PACKAGES` to `TRUE` or `1` to enable, `FALSE` or `0` to
```

Comment:
It's annoying, but I think we should allow literally `"true"` or `"false"`, or the whole range of `TRUE`, `"TRUE"`, "true"`, `"1"` and `1` and false variants. (Basically, `tolower(val) %in% c("true", "1").)

I would read this expecting I could set `SHINYLIVE_WASM_PACKAGES=TRUE` in `.Renviron`, but IIRC that will end up being `"TRUE"` when set.

Reply:
I think this should be handled OK by:

```r
  pkgs_env <- switch(pkgs_env, "1" = TRUE, "0" = FALSE, pkgs_env)
  wasm_packages <- as.logical(pkgs_env)
```

Because "1" will be caught by the `switch`, and then `"TRUE"`/`"true" will be caught by the `as.logical()`.

```r
> as.logical("TRUE")
[1] TRUE
> as.logical("true")
[1] TRUE
> as.logical(TRUE)
[1] TRUE
> as.logical("false")
[1] FALSE
> as.logical("something-weird")
[1] NA
```

Saying that, I might change it just to make things clearer.

Reply:
oh nice, that makes sense. I just missed that `as.logical()` line